CHANGELOG.md

```diff
@@ -8,6 +8,8 @@
 
 ## Improvements
 
+- PR #1076 Use `type_dispatcher` in join, quantiles, filter, segmented sort, radix sort and hash_groupby
+- PR #1076 Added type trait to get underlying type of wrapped type
```

Comment:
Please don't add two lines for the same PR. I don't think the changelog has to list every single change you made, so you can delete this second line.